### PR TITLE
rake: init at 11.1.1

### PR DESCRIPTION
**Pull request by namore**
Add package "rake", which is a ruby-based build tool.
Obviously, no other packages depend on rake, so the addition should not break anything.
###### Things done:
- [ ] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"` (... no dependencies)
- [X] Tested execution of all binary files (usually in `./result/bin/`) (... no binary files)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by joachifm** [comment]
The desired commit subject format for new packages is `pkgname: init at version`.

File: pkgs/development/tools/build-managers/rake/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+bundlerEnv {
+  name = "rake-11.1.1";
+
+  inherit ruby;
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+  
+  meta = with lib; {
+    description = "A software task management and build automation tool.";
```
Comment (joachifm) [inline pkgs/development/tools/build-managers/rake/default.nix]: Please remove the trailing dot from the description.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -9302,6 +9302,10 @@ let
 
   radius = callPackage ../servers/radius { };
 
+  rake = callPackage ../development/tools/build-managers/rake {
+#    ruby = ruby_2_1;
```
Comment (joachifm) [inline pkgs/top-level/all-packages.nix]: Unless this is intended to serve as documentation of some sort, please remove it.

**Comment by namore** [comment]
Thanks for your feedback, @joachifm. I made the requested changes.
By the way, the [Ruby example](https://hydra.nixos.org/build/33390848/download/1/nixpkgs/manual.html#sec-language-ruby) in the nixpkgs manual contains a trailing dot in its description.
This may confuse newcomers such as me.

**Comment by joachifm** [comment]
I fixed the usage example. Thanks.

**Comment by joachifm** [comment]
Err, I didn't notice this was against release 15.09 ...

**Comment by namore** [comment]
@joachifm sorry if pull request was against wrong branch. I am still not quite sure which branch is best, staging or master perhaps? Will the package be available in the "15.09 channel" if it is merged into master or staging?

**Comment by joachifm** [comment]
master is the preferred branch. staging is for non-breaking changes that cause large rebuilds, it exists mainly to let the build farm "catch up" and so minmize disruption for people running the unstable version.

A change like this wouldn't ordinarily be integrated into a release branch, they typically only see bug and security fixes once the release has been finalized. If there is a specific reason you think it should be integrated into a release branch, there's no harm in requesting it, of course.

Please re-open the PR against master.

**Comment by namore** [comment]
@joachifm Thank you for the good explanation. :-) As suggested, I will re-open the PR against master.

**Pull request merged**